### DISCUSSION TITLE
예약성공해도 네이버 예약로그에는 안남을 수 있습니다.

### DISCUSSION THREAD

**Discussion by max-jang** [changdoc/naver-vaccine-macro#46] [General]
예약성공해도 네이버 예약로그에는 안남을 수 있습니다. 

저번주에 지인 해드렸는데 실패라고 뜨지만(잔여백신 마감) 몇분뒤에 문자로 예약확정 수신하였습니다.

아마 질관청 전산에는 요청이 갔는데 네이버 쪽에서 쪽나서 그렇게 날라올 수도 있겠거니 싶거군요
네이버 예약기록에도 실패나 성공 두개 다 안뜹니다.

실패해도 국민비서 SMS 메시지/병원으로 오는 전화 한번씩 확인해보세요!

**Comment by jj74523** [comment]
오 그렇군요 정보 감사합니다~
